### PR TITLE
15951 - Special Resolution Correction - QA / UX fixes

### PR DESCRIPTION
**Pull request by seeker25**
*Issue #:* /bcgov/entity#15951

*Description of changes:*
Special Resolution Correction - QA / UX fixes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/mixins/filing-template-mixin.ts [hunk 1]
```diff
@@ -766,7 +766,12 @@ export default class FilingTemplateMixin extends DateMixin {
 
     if (this.isCoopCorrectionFiling) {
       this.storeSpecialResolutionRulesAndMemorandum(filing.correction, entitySnapshot)
-      const specialResolution = this.getLatestResolutionForBusiness
+      let specialResolution: SpecialResolutionIF = {}
+      if (filing.correction.resolution) {
+        specialResolution = filing.correction
```
Comment (seeker25) [inline src/mixins/filing-template-mixin.ts:771]: Need for for draft

**Review by Jxio — APPROVED** [review]

File: src/mixins/filing-template-mixin.ts (hunk 1 above)
Comment (severinbeauvais) [reply src/mixins/filing-template-mixin.ts:771]: It seems weird to assign the entire `correction` object to `specialResolution`. I thought `correction` would contain `specialResolution` (or some of its properties), but this makes them seem equivalent...?

Should it look like this?
```
specialResolution = {
  meetingDate: filing.correction.meetingDate,
  resolution: filing.correction.resolution,
  resolutionDate: filing.correction.resolutionDate,
  <etc>
}
```

Comment (severinbeauvais) [reply src/mixins/filing-template-mixin.ts:771]: By the way, I think there's a typing error on line 145.

**Comment by pwei1018** [comment]
Temporary Url for review: https://business-edit-dev--pr-509-nje1txxx.web.app

File: src/mixins/filing-template-mixin.ts (hunk 1 above)
Comment (Jxio) [reply src/mixins/filing-template-mixin.ts:771]: yes correction includes much more properties, I'll update it

**Comment by pwei1018** [comment]
Temporary Url for review: https://business-edit-dev--pr-509-nje1txxx.web.app

**Review by severinbeauvais — APPROVED** [review]
LGTM.

**Comment by seeker25** [comment]
working on unit tests

**Comment by pwei1018** [comment]
Temporary Url for review: https://business-edit-dev--pr-509-nje1txxx.web.app

**Pull request merged**